CryptoTechReminderSystem/UseCase/GetUsers.cs

```diff
@@ -0,0 +1,19 @@
+using CryptoTechReminderSystem.Gateway;
+
+namespace CryptoTechReminderSystem.UseCase
+{
+    public class GetUsers
+    {
+        private HarvestGateway _requester;
+
+        public GetUsers(HarvestGateway harvestRequester)
+        {
+            _requester = harvestRequester;
+        }
+
+        public string Execute()
+        {
+            return "asdfasdf";
```

Comment:
That value doesn't look right

Reply:
Yes, this file probably did not need committing in this PR.